src/app/components/pages/myList.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+
+
+export default function MyList() {
```

Comment:
MyList nie wiadomo o co chdzi. Co to za komponent?

Reply:
to podstrona, gdzie bedzie lista dodanych filmow przez uzytkownika